docs/source/notebooks/pcca.ipynb

```diff
@@ -96,29 +177,136 @@
+    "    ax = axes[i]\n",
+    "    ax.set_title(f\"Metastable set {i+1} assignment probabilities\")\n",
+    "    \n",
+    "    ax.scatter(*data.T, c=pcca.memberships[assignments, i], cmap=plt.cm.Blues)\n",
+    "norm = mpl.colors.Normalize(vmin=0, vmax=1)\n",
+    "fig.colorbar(plt.cm.ScalarMappable(norm=norm, cmap=plt.cm.Blues), ax=axes, shrink=.8);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As expected, the assignments nicely disentangle the two ellipsoids.\n",
     "\n",
-    "f, ax = plt.subplots(1, 1, figsize=(12, 12))\n",
-    "ax.scatter(*data.T, c=assignments)\n",
-    "ax.scatter(*clustering.cluster_centers.T, marker=\"*\", \n",
-    "            c=['red' if m == 0 else 'cyan' for m in crisp_memberships])"
+    "One as also access to a coarse-grained stationary probability vector"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pcca.coarse_grained_stationary_probability"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Another offered quantity are the metastable distributions, i.e., the probability of metastable states to visit a micro state by PCCA+ $\\mathbb{P}(\\text{state}_{\\text{micro}} | \\text{state}_\\text{pcca})$:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Metastable distributions shape:\", pcca.metastable_distributions.shape)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(10, 5))\n",
+    "plt.plot(np.arange(msm.n_states)+1, pcca.metastable_distributions[0], 'o', label=\"Metastable set 1\")\n",
+    "plt.plot(np.arange(msm.n_states)+1, pcca.metastable_distributions[1], 'x', label=\"Metastable set 2\")\n",
+    "plt.xticks(np.arange(msm.n_states)+1)\n",
+    "plt.title(\"Metastable distributions\")\n",
+    "plt.legend()\n",
+    "plt.xlabel('micro state')\n",
+    "plt.ylabel('probability');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For all micro states one of the distribution values is approximately zero whereas the other one is greater than zero. This is because micro states were separated well enough in the coarse-graining."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For visualization purposes, one can obtain the crisp assignments of microstates to macrostates through:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "pcca.assignments"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "However caution is appropriate, one *cannot* compute any actual quantity of the coarse-grained kinetics without employing the fuzzy memberships."
```

Comment:
probably worth mentioning that the original PCCA formulation was hard memberships? we can discuss/i can give you citations and stuff

Reply:
original is deuflhard lin alg appl 2000

Reply:
mentioned in introduction now